main.js

```diff
@@ -7,6 +7,20 @@ const BrowserWindow = electron.BrowserWindow;
 const path = require('path');
 const url = require('url');
 const fs = require('fs');
+const winston = require('winston')
```

Comment:
I know it's outside of the scope of conversion but I think we should just bite the bullet and semicolon fix all the js files in this pass. Thoughts on this?

Reply:
Fine with me! Eslint should make that easy. I'm not clear on what the desired style is, though; do you want semicolons required or semicolons forbidden?

Reply:
Required semicolons for sure :)